### DISCUSSION TITLE
How do I select font-weight in troika-three-text?

### DISCUSSION THREAD

**Discussion by Ashvith10** [protectwise/troika#277] [Q&A] [answered]
I want to use the weight "black" for Cantarell, but I'm unable to figure out how I  can do the same...

**Comment by lojjic** [comment, marked as answer]
You need to use the font file that contains that weight.

**Reply by Ashvith10** [reply]
@lojjic thanks for answering. For some reason, I'm not able to grasp this: why does the font Cantarell have five different states of weight, despite having only two sizes (Regular and Bold) normally? I am not able to make my text thicker than what's rendered normally on the HTML page?

Size 100 and 200
![Screen Shot 2023-08-29 at 04 18 37](https://github.com/protectwise/troika/assets/113123021/d4fb89c1-e227-4a86-b68d-490ea9360172)

Size 300
![Screen Shot 2023-08-29 at 04 18 51](https://github.com/protectwise/troika/assets/113123021/7dc23766-c648-4386-b305-a51ed468519e)

Size 400 and 500
![Screen Shot 2023-08-29 at 04 18 51](https://github.com/protectwise/troika/assets/113123021/42dcc03e-bf89-4a78-8d06-c77aa0d0cef2)

Size 600 and 700
![Screen Shot 2023-08-29 at 04 19 20](https://github.com/protectwise/troika/assets/113123021/f0c2d00c-f728-4d41-a4b9-556daa6bbfd6)

Size 800 and 900
![Screen Shot 2023-08-29 at 04 19 30](https://github.com/protectwise/troika/assets/113123021/4837a7b3-0169-4c9c-96f0-c765aa2fad25)

Meanwhile, for the 3D text:
Regular
![Screen Shot 2023-08-29 at 04 26 53](https://github.com/protectwise/troika/assets/113123021/78dc5801-94de-474c-aa7e-850a7a758255)

Bold (700)
![Screen Shot 2023-08-29 at 04 22 17](https://github.com/protectwise/troika/assets/113123021/711cd420-30b4-46f6-9872-00647cb0c99c)

How can I access the other states of weight?

**Reply by lojjic** [reply]
I don't think I have enough context to answer your question. I only see two weights for Cantarell on google fonts. Is some other tool showing you intermediate weights?

**Reply by Ashvith10** [reply]
You're right @lojjic , there are only two weights. But HTML can apparently create five magically. Well, I wanted to understand if I could also do the same with just two files for a 3D text?

**Reply by lojjic** [reply]
Ah, well, the answer to whether it can do that sort of magic is no. ;)

I'm unaware of that magic though. A quick test changing the font-weight with devtools on https://fonts.google.com/specimen/Cantarell doesn't show anything but the two main weights. I tested Firefox, Safari, and Chrome on Mac.

**Reply by Ashvith10** [reply]
My previous comments were a little misleading. The browser is using the Cantarell Regular font (to be entirely accurate, `system-ui`), and doing some magic behind the scene.

![Screenshot from 2023-08-29 06-30-12](https://github.com/protectwise/troika/assets/113123021/57a908fd-6280-496c-9188-186652f9e13d)

There's no involvement of Cantarell Bold here. All of this has to do with algorithmic bolding. This behavior is very inconsistent between different browsers. Perhaps I'll look for a font that has all the weights available.
